Converters/Autocad/Speckle.Converters.AutocadShared/ToSpeckle/Raw/DBSplineToSpeckleRawConverter.cs

```diff
@@ -107,17 +107,14 @@ public SOG.Curve Convert(ADB.Spline target)
       rational = target.IsRational,
       closed = periodicClosed || target.Closed,
       length = length,
-      domain = domain,
+      domain = domain ?? null!, //TODO: what should we do here?
```

Comment:
We may want to double check this as this code is part of a "hack" with a POC comment

Reply:
I've resolved it with some input from Claire, now defaulting to the `UnitInterval`